### ISSUE TITLE
No file or variants found for asset: images/diamond.png

### ISSUE THREAD

**Issue by Orlando00** [londonappbrewery/Flutter-Course-Resources#35]
I keep getting this error code. My indents look correct in YAML. Not sure what's going on. Thanks

![Annotation 2020-08-11 095621](https://user-images.githubusercontent.com/69528297/89926010-18d0ef00-dbb9-11ea-9a95-74381e4e14bb.png)

**Comment by Orlando00** [comment]
I went into YAML and changed it to : 

 assets:
    - images/

That runs the app now but wont show the diamond. 

I/flutter (12936): Unable to load asset: images/diamond.png

Is the error I'm getting.

**Comment by fvaladares** [comment]
Hi.

Did you download the Diamond file from the resources and save it inside the
folder Images in your project?

Regards
---
*Fabrício G. Valadares*
Lattes: http://lattes.cnpq.br/1002632563427298
Mobile: +55 31 9 9357-6193
Linkedin:http://br.linkedin.com/pub/fabricio-valadares/45/586/5a1/
mail: fabricio.valadares@gmail.com

*Os homens conseguiriam muito mais coisas se julgassem menos coisas
impossíveis.*

Embora ninguém possa voltar atrás e fazer um novo começo, qualquer um pode
começar agora e fazer um novo fim. Francisco Cândido Xavier


The information contained in this email and any attachments is confidential
and may be subject to copyright or other intellectual property protection.
If you are not the intended recipient, you are not authorized to use or
disclose this information, and we request that you notify us by reply mail
or telephone and delete the original message from your mail system.

**Comment by Orlando00** [comment]
Yes I did. I can see it in the file folder and when I double click it, it comes on the screen.

**Comment by Orlando00** [comment]
I got it to work. I exited out of Android Studios and brought back up and now it works lol

**Comment by fvaladares** [comment]
Sometimes, when you make changes on your project it's interesting to run
the command File/Invalidate cache/restart -> It solves a lot of problems :)

;)
---
*Fabrício G. Valadares*
Lattes: http://lattes.cnpq.br/1002632563427298
Mobile: +55 31 9 9357-6193
Linkedin:http://br.linkedin.com/pub/fabricio-valadares/45/586/5a1/
mail: fabricio.valadares@gmail.com

*Os homens conseguiriam muito mais coisas se julgassem menos coisas
impossíveis.*

Embora ninguém possa voltar atrás e fazer um novo começo, qualquer um pode
começar agora e fazer um novo fim. Francisco Cândido Xavier


The information contained in this email and any attachments is confidential
and may be subject to copyright or other intellectual property protection.
If you are not the intended recipient, you are not authorized to use or
disclose this information, and we request that you notify us by reply mail
or telephone and delete the original message from your mail system.

**Comment by Scibud** [comment]
Hi check [this issue](https://github.com/flutter/flutter/issues/11199) for clearing the doubts. 

Indentation matters in Flutter

**Comment by prateek-aher** [comment]
@Orlando00 Close this issue.
